lib/ruby_indexer/lib/ruby_indexer/configuration.rb

```diff
@@ -43,6 +50,15 @@ def initialize
       )
     end
 
+    sig { returns(String) }
+    def exclude_pattern
+      relative_patterns = @excluded_patterns
+        .select { |p| p.end_with?("/**/*") }
+        .map { |p| p.delete_prefix("#{Dir.pwd}/") }
+
+      "#{Dir.pwd}/{#{relative_patterns.join(",")}}"
+    end
```

Comment:
Why do we need this method? Could we instead change `@excluded_patterns` to always begin with `Dir.pwd`? Or is the idea to guarantee that user added patterns also start with `Dir.pwd`?

Reply:
This method returns a single string. I have a test for it based on the `@excluded_patterns` in line 23. Perhaps it could use a comment. Check this test to see the computed output:

https://github.com/Shopify/ruby-lsp/pull/2082/files#diff-fd9cbb60b18cc9e588f3863d60dfcca34ff90e6cb488169105c973fbaf8eaba8R55